src/compat/db_json.cpp

```diff
@@ -1204,7 +1204,7 @@ db_json_value_is_contained_in_doc_helper (const JSON_VALUE *doc, const JSON_VALU
 
   doc_type = db_json_get_type_of_value (doc);
   val_type = db_json_get_type_of_value (value);
-  result = false;
+  result = true;
```

Comment:
Since the initialization affects only the case of `else if (doc_type == DB_JSON_OBJECT)`, I'd like to propose to remove the initialization here and move it to the block to make it clearer. 
Comment to describe the behavior (empty is regarded as included) would be good.